wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -96,6 +97,12 @@ extension ConversationListViewController {
             super.init()
 
             updateE2EICertifiedStatus()
+            notificationCenter.addObserver(
+                self,
+                selector: #selector(handleApplicationDidBecomeActiveNotification(_:)),
+                name: UIApplication.didBecomeActiveNotification,
+                object: nil
+            )
```

Comment:
Personally I would rather use the token api instead of selectors with `@objc` annotations...
```swift
        deinit {
            if let observerToken {
                notificationCenter.removeObserver(observerToken)
            }
        }

        private func setupObservers() {
            observerToken = notificationCenter.addObserver(
                forName: UIApplication.didBecomeActiveNotification,
                object: nil,
                queue: .main,
                using: { _ in
                    Task { [weak self] in
                        await self?.updateE2EICertifiedStatus()
                    }
                }
            )
        }
```